lib/plugins/aws/lib/naming.js

```diff
@@ -49,6 +49,22 @@ module.exports = {
     return `${this.provider.serverless.service.service}-${this.provider.getStage()}`;
   },
 
+  getServiceArtifactName() {
+    return `${this.provider.serverless.service.service}.zip`;
```

Comment:
Does this cover the case for Java projects, where the package artifact is a jar file?  I think the current functionality allows users to specify the artifact name under the package:artifact section of serverless.yml.  I for one need and use this functionality.